var/spack/repos/builtin/packages/python/package.py

```diff
@@ -1127,3 +1127,22 @@ def remove_files_from_view(self, view, merge_map):
                 view.remove_file(src, dst)
             else:
                 os.remove(dst)
+
+    def test(self):
+        # do not use self.command because we are also testing the run env
+        exe = self.command.name
+
+        self.run_test('/bin/false')
+
+        self.run_test('/usr/bin/false')
+
+        self.run_test('/usr/bin/true', status=1)
```

Comment:
What is the point of this?

Reply:
This was debugging code --  we should remove it before merging.

Reply:
Removing